spec/index.html

```diff
@@ -740,28 +740,25 @@ <h3>Literals</h3>
       <li>If the literal is a <a>language-tagged string</a>,
         then the literal value is a pair consisting of its <a>lexical form</a>
         and its <a>language tag</a>, in that order.</li>
-      <li>if the literal is a <a>directional language-tagged string</a>, then the literal value is
+      <li>If the literal is a <a>directional language-tagged string</a>, then the literal value is
         a tuple of its <a>lexical form</a>, its <a>language tag</a>, and its <a>base direction</a>,
         likewise in that order.</li>
-
-      <li>If the literal's <a>datatype IRI</a> is in the set of
-        <a>recognized datatype IRIs</a>, let <var>d</var> be the
-        <a>referent</a> of the datatype IRI.
+      <li>If the literal's <a>datatype IRI</a> is handled by an RDF processor,
         <ul>
-          <li>If the literal's <a>lexical form</a> is in the <a>lexical space</a>
-            of <var>d</var>, then the literal value is the result of applying
-            the <a>lexical-to-value mapping</a> of <var>d</var> to the
+          <li>if the literal's <a>lexical form</a> is in the <a>lexical space</a>
+            of datatype, then the literal value is the result of applying
+            the <a>lexical-to-value mapping</a> of the datatype to the
             <a>lexical form</a>.</li>
-          <li>Otherwise, the literal is ill-typed and no literal value can be
+          <li>otherwise, the literal is ill-typed and no literal value can be
              associated with the literal. Such a case produces a semantic
              inconsistency but is not <em>syntactically</em> ill-formed.
-             Implementations MUST accept ill-typed literals and produce RDF
+             Implementations SHOULD accept ill-typed literals and produce RDF
              graphs from them. Implementations MAY produce warnings when
              encountering ill-typed literals.</li>
         </ul>
       </li>
-      <li>If the literal's <a>datatype IRI</a> is <em>not</em> in the set of
-        <a>recognized datatype IRIs</a>, then the literal value is
+      <li>If the literal's <a>datatype IRI</a> is <em>not</em>
+        handled by an RDF processor then the literal value is
```

Comment:
```suggestion
        handled by the implementation, then the literal value is
```
(see my comment above about "an RDF processor")

Reply:
I've pushed changes that include changing "RDF processor" to "RDF implenentation".